ports/libsimpleservo/capi/Cargo.toml

```diff
@@ -15,6 +15,7 @@ bench = false
 [dependencies]
 simpleservo = { path = "../api" }
 log = "0.4"
+lazy_static = "1"
```

Comment:
This now adds a duplicate key and the result doesn't parse as valid TOML. It will need to be removed.

Reply:
Yes, I didn't do a pull the latest changes from servo's master.  Working on the fix now.

Reply:
Humm….
If I remove lazy_static from the capi\Cargo.toml I get:

```
C:\git-mozilla\servo-angelortiz1007>mach build -d --uwp
   Compiling simpleservo_capi v0.0.1 (C:\git-mozilla\servo-angelortiz1007\ports\libsimpleservo\capi)
error[E0464]: multiple matching crates for `lazy_static`
 --> ports\libsimpleservo\capi\src\lib.rs:6:1
  |
6 | extern crate lazy_static;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: candidates:
          crate `lazy_static`: \\?\C:\Users\Angel\.rustup\toolchains\nightly-2019-07-31-x86_64-pc-windows-msvc\lib\rustlib\x86_64-pc-windows-msvc\lib\liblazy_static-2f02f603e2ac1559.rlib
          crate `lazy_static`: \\?\C:\Users\Angel\.rustup\toolchains\nightly-2019-07-31-x86_64-pc-windows-msvc\lib\rustlib\x86_64-pc-windows-msvc\lib\liblazy_static-4c805e4a59b12c5b.rlib

error[E0463]: can't find crate for `lazy_static`
 --> ports\libsimpleservo\capi\src\lib.rs:6:1
  |
6 | extern crate lazy_static;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0463`.
error: Could not compile `simpleservo_capi`.
```

Reply:
It's present on master, so you need to pull in the latest changes first.